app/assets/javascripts/datatables.js.coffee

```diff
@@ -1,2 +1,4 @@
 $(document).ready ->
-  $('.data_table').DataTable()
+  $('.data_table').DataTable({
+    'lengthMenu': [[10, 25, 50, -1], [10, 25, 50, 'All']]
+    });
```

Comment:
Keep in mind that this is CoffeeScript, hence the CC error.

This could be one line:

```coffee
$('.data_table').DataTable lengthMenu: [[10, 25, 50, -1], [10, 25, 50, 'All']]
```